fundingmanager.go

```diff
@@ -77,6 +77,23 @@ type reservationWithCtx struct {
 	err     chan error
 }
 
+// isLocked checks the reservation's timestamp to determine whether it is locked.
+// The time zero value represents a locked reservation
```

Comment:
nit: period. 

Also this last sentence probably can be moved into the method, as the caller doesn't need to know this.

Reply:
done